src/MTGCore.ClientApp/ClientApp/src/components/SearchFilterModal.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <transition name="modal" v-if="ShowModal">
+    <div
+      class="modal"
+      id="TitleModal"
+      style="display: block; background-color: rgba(0, 0, 0, 0.5); transition: opacity 0.3s ease"
+    >
```

Comment:
Nitpick: Is this prettier or something? Not sure I am a fan of the chevron on it's own line.